src/soundio/soundmanager.h

```diff
@@ -107,7 +108,7 @@ class SoundManager : public QObject {
     QList<AudioOutput> registeredOutputs() const;
     QList<AudioInput> registeredInputs() const;
 
-    QSharedPointer<EngineNetworkStream> getNetworkStream() const {
+    const std::unique_ptr<EngineNetworkStream>& getNetworkStream() const {
```

Comment:
In general: Don't use references to unique pointers. Use raw pointers if the ownership is not passed and std::move if the ownership is passed.